src/ui/style/palette.scss

```diff
@@ -16,6 +17,7 @@ $secondary-disabled: #b2c3d1;
 // neutral (grey)
 $neutral: #babec2;
 $neutral-variant-1: #dfe4e9;
+$neutral-900: #4A4C4E;
```

Comment:
This variables break the current palette design, even if the OKP4 design system has been updated accordingly.
I think we should rework the whole palette to take in account the changes in a very next future

Reply:
Definitely agree with your concern. @ccamel recently provided a similar [feedback](https://github.com/okp4/dataverse-portal/pull/461#discussion_r1317194833).

A [proposal](https://github.com/okp4/dataverse-portal/discussions/462) has been created to address the issue. Are you OK to let it as is and provide a solution in an dedicated PR?

Reply:
I will say no, because the risk is to not handle your suggestion in the discussion, mainly because of lack of time.
And then, we will keep an incorrect and degraded version of the `palette` and, by extension, the understanding of the whole thematization..
Please, revert these variables and let's improve it in a dedicated refund PR. Thanks 🙏